README.rst

```diff
@@ -118,13 +118,40 @@ DNS lookup...
 * If ``gethostbyname`` does not have it cached nor in the ``hosts`` file then a request
   is made to the known DNS server that was given to the network stack. This is typically the
   local router or the ISP's caching DNS server.
+  OnWindows it will use an algorithm to determine which DNS server to try first
+  to resolve the hostname. http://technet.microsoft.com/en-us/library/dd197552(WS.10).aspx
 * The local DNS server (or local gateway's) MAC address is looked up in the ARP
   cache. If the MAC address is missing, an ARP request packet is sent.
 * Port 53 is opened to send a UDP request to DNS server (if the response size is
```

Comment:
As far as I know, port 53 is not open on the DNS client, only on the server. See [RFC 1035](https://datatracker.ietf.org/doc/html/rfc1035#:~:text=Messages%20sent%20using%20UDP%20user%20server%20port%2053%20(decimal))
In fact, random client ports help preventing spam and spoofing as the DNS server have to reply on the same port.